src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -45,72 +47,90 @@
+           (when (and customization-color (not= :watch-only type))
+             [customization-colors/overlay
+              {:customization-color customization-color
+               :border-radius       16
+               :theme               theme
+               :pressed?            @pressed?}])
+           [rn/view {:style style/profile-container}
+            [rn/view {:style {:padding-bottom 2 :margin-right 2}}
+             [text/text {:style style/emoji} emoji]]
+            [rn/view {:style style/watch-only-container}
              [text/text
-              {:weight :semi-bold
-               :size   :paragraph-2
-               :style  (style/metrics watch-only? theme)} account-amount]
-             [rn/view {:style {:margin-left 4}}
-              [icon/icon :positive
-               {:color (colors/theme-colors (if watch-only? colors/neutral-50 colors/white-opa-70)
-                                            colors/white-opa-70
-                                            theme)
-                :size  16}]]])])])))
+              {:size   :paragraph-2
+               :weight :medium
+               :style  (style/account-name watch-only? theme)}
+              account-name]
+             (when watch-only? [icon/icon :reveal {:color colors/neutral-50 :size 12}])]]
+           [text/text
+            {:size   :heading-2
+             :weight :semi-bold
+             :style  (style/account-value watch-only? theme)}
+            balance]
+           (when metrics?
+             [rn/view {:style style/metrics-container}
+              [text/text
+               {:weight              :semi-bold
+                :size                :paragraph-2
+                :accessibility-label :metrics
+                :style               (style/metrics watch-only? theme)}
+               account-percentage]
+              (when (not empty-type?)
+                [:<>
+                 [rn/view (style/separator watch-only? theme)]
+                 [text/text
+                  {:weight :semi-bold
+                   :size   :paragraph-2
+                   :style  (style/metrics watch-only? theme)} account-amount]
+                 [rn/view {:style {:margin-left 4}}
+                  [icon/icon :positive
+                   {:color (colors/theme-colors (if watch-only? colors/neutral-50 colors/white-opa-70)
+                                                colors/white-opa-70
+                                                theme)
+                    :size  16}]]])])])))))
 
-(defn- add-account-view
-  [{:keys [on-press customization-color theme metrics?]}]
-  [rn/view (style/add-account-container theme metrics?)
-   [button/button
-    {:type                :primary
-     :size                24
-     :icon                true
-     :accessibility-label :add-account
-     :on-press            on-press
-     :customization-color customization-color
-     :icon-only?          true}
-    :i/add]])
+(def add-account-view
```

Comment:
This should be `defn` instead of `def`. Comment with more explanation https://github.com/status-im/status-mobile/pull/17234/files#r1319930535

Reply:
Done